mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
Yeah it would mess things up, since it would disconnect _conn. It's a private function so hopefully it won't be called outside of the expected lifecycle but I can wrap it in a check for self._conn == null if you like. lmk what you want to see.